ADAL/src/public/ADAL.h

```diff
@@ -26,6 +26,21 @@ FOUNDATION_EXPORT double ADALFrameworkVersionNumber;
 //! Project version string for ADALiOSFramework.
 FOUNDATION_EXPORT const unsigned char ADALFrameworkVersionString[];
 
+#if TARGET_OS_IPHONE
+//iOS:
```

Comment:
We don't, Apple does. That's a system macro. It dates back to when iOS was called "iPhone OS"